bravado/client.py

```diff
@@ -174,8 +174,8 @@ def __getattr__(self, name):
 
     def construct_request(self, **op_kwargs):
         """
-        :param op_kwargs: parameter name/value pairs to pass to the invocation
-            of the operation.
+        :param op_kwargs: parameter name/value pairs to passed to the
+            invocation of the operation.
         :return: request in dict form
         """
         request_options = op_kwargs.pop('_request_options', {})
```

Comment:
Will create another PR to make sure `_request_options` and `headers` are not mutated.